content/changelog/2024-11-15.md

```diff
@@ -0,0 +1,67 @@
+
+And don't forget to check out this week's fixes and improvements:
+
+<details>
+
+<summary>**Fixes & improvements**</summary>
+
+- **Collation support**
+
+  By default, Neon now uses the `C.UTF-8` collation, which supports the full range of UTF-8 encoded characters. Previously, Neon used the `C` collation provided by `libc` by default. For more about collation support in Neon, see [Collation support](/docs/reference/compatibility#collation-support).
+
+- **Neon API updates**
+
+  - Improved validation of project ID lists with the [Get consumption metrics for each project](https://api-docs.neon.tech/reference/getconsumptionhistoryperproject) endpoint. Previously, restrictive validation caused the endpoint to fail for some users.
+
+- **Neon CLI enhancements**
+
+  The Neon CLI was updated to version 2.4.0. For upgrade instructions, see [Upgrading the Neon CLI](https://neon.tech/docs/reference/cli-install#upgrade).
+
+  The `branches list` command now shows a branch's `Current State`. Branch states include:
+
+  - `init` - the branch is being created but is not yet available for querying.
+  - `ready` - the branch is fully operational and ready for querying. Expect normal query response times.
+  - `archived` - the branch is stored in cost-effective archive storage. Expect slow query response times.
+
+    ```bash
+    neon branches list --project-id green-hat-46829796
+    ┌───────────────────────────┬──────┬─────────┬───────────────┬──────────────────────┐
+    │ Id                        │ Name │ Default │ Current State │ Created At           │
+    ├───────────────────────────┼──────┼─────────┼───────────────┼──────────────────────┤
+    │ br-muddy-firefly-a7kzf0d4 │ main │ true    │ ready         │ 2024-10-30T14:59:57Z │
+    └───────────────────────────┴──────┴─────────┴───────────────┴──────────────────────┘
+    ```
+
+  The `Updated At` value was removed from the `branches list` command output. This value reflected internal metadata changes only and provided limited value.
+
+- **Drizzle Studio update**
+
+  The Drizzle Studio integration that powers the **Tables** page in the Neon Console has been updated to version 0.0.20. For improvements and fixes in this version, see the [Neon Drizzle Studio Integration Changelog](https://github.com/neondatabase/neon-drizzle-studio-changelog/blob/main/CHANGELOG.md).
+
+- **Fixes**
+
+  - Fixed an issue where users who were removed from an organiation got an error page when logging in to Neon. The console was incorrectly redirecting them to the organization page, which they can no longer access. Users are now directed to their personal account **Projects** page instead.
+  - When you make changes to your first or last name in **Account Settings**, those changes are now immediately reflected. Previously, old values could sometimes persist until the page was reloaded.
+
+    <Admonition type="tip" title="Did you know?">
```

Comment:
maybe bring this out of the list of fixes? Last week it was related to the item listed above it. wdyt?